api/reporting/schemas/input/mysql_describe_csv/user.csv

```diff
@@ -0,0 +1,47 @@
+user_id,bigint(20),NO,PRI,,auto_increment
```

Comment:
Should the csv/text files be generated somehow, instead of committing them into the project. Won't the files become stale over time as the schema evolves?

Reply:
Eh, there's some awkwardness here, and I'm trying to resist over-building this tool. In general, adding a new column to an app DB table does not mean it needs to show up in the reporting dataset. They're only 90% the same, so there is still some manual fixup. 

I think a good compromise would be to remove this input directory upon scripting the sql calls, but not before. That's not something that needs to happen just yet IMO, as it doesn't affect my stakeholders.